package/utils/busybox/patches/220-add_lock_util.patch

```diff
@@ -74,7 +74,7 @@
 +{
 +	int pid;
 +	int flags;
-+	char pidstr[8];
++	char pidstr[10];
```

Comment:
I would prefer 11 here to also store the termination NULL byte.
INT_MAX is 2147483647

Reply:
Please also change the place where this buffer is used to
``
snprintf(sizeof(pidstr), pidstr, "%d\n", pid);
``